test/tests/mesh/checkpoint/tests

```diff
@@ -1,33 +1,57 @@
 [Tests]
```

Comment:
Let's just remove the duplicate tests. These requirements aren't particularly useful, anyway. Testing this with just a single set of procs is fine. Requirements should be general enough s.t. they don't list number of procs